tests/VolleyManagement.Specs/PlayersContext/CreatePlayer.feature

```diff
@@ -3,6 +3,11 @@
 	I want to be able to create players
     So they can be assigned to teams
 
+Scenario: Create very long name
+Given first name set to Very looong name which should be more than 60 symbols
+When I execute CreatePlayer
+Then InvalidEntityException is thrown
```

Comment:
@sdiachen 
because it's real life situation. You want EntityInvariantVoiolationException or AgrumentException but it has no sense. When we create player we don't use Player.cs class which  has this exception. We operate with CreatePLayerDto and check is it valid only here.
![image](https://user-images.githubusercontent.com/28676005/39717369-bcd0541a-523b-11e8-9e85-32b10faf4179.png)

So there 2 options. Leave as it is or change CreatePlayerDto by adding validation logic to this.

Reply:
OK